METRICS.md

```diff
@@ -70,47 +77,54 @@ memory_stats/swap_usage/failcnt | uint64 | The number of times the swap space li
+memory_stats/statistics/total_mapped_file | uint64 | The total number of bytes of mapped file (includes tmpfs/shmem) <sup>(1)</sup>
+memory_stats/statistics/total_pgfault | uint64 | The total number of page faults which happened since the creation of the cgroup <sup>(1)</sup>
+memory_stats/statistics/total_pgmajfault | uint64 | The total number of page major faults which happened since the creation of the cgroup <sup>(1)</sup>
+memory_stats/statistics/total_pgpgin | uint64 | The total number of charging events to the memory cgroup <sup>(1)</sup>
+memory_stats/statistics/total_pgpgout | uint64 | The total number of uncharging events to the memory cgroup <sup>(1)</sup>
+memory_stats/statistics/total_rss | uint64 | The total number of bytes of anonymous and swap cache memory <sup>(1)</sup>
+memory_stats/statistics/total_rss_huge | uint64 | The total number of bytes of anonymous transparent hugepages <sup>(1)</sup>
+memory_stats/statistics/total_swap | uint64 | The total number of bytes of swap usage <sup>(1)</sup>
+memory_stats/statistics/total_unevictable | uint64 | The total number of bytes of memory that cannot be reclaimed <sup>(1)</sup>
+memory_stats/statistics/total_writeback | uint64 | The total number of bytes of file/anon cache that are queued for syncing to disk <sup>(1)</sup>
+| |
+hugetlb_stats/\<size\>/failcnt | uint64 | The number of allocation failure due to HugeTLB limit
+hugetlb_stats/\<size\>/max_usage | uint64 | Max "hugepagesize" hugetlb  usage recorded
+hugetlb_stats/\<size\>/usage | uint64 | Current usage for "hugepagesize" hugetlb
+| |
+blkio_stats/io_merged_recursive/\<N\>/value | uint64 | Total number of bios/requests merged into requests belonging from all the descendant cgroups 
+blkio_stats/io_queue_recursive/\<N\>/value | uint64 | Total number of requests queued up at any given instant from all the descendant cgroups
+blkio_stats/io_service_bytes_recursive/\<N\>/value | uint64 | Number of bytes transferred to/from the disk from all the descendant cgroups
+blkio_stats/io_service_time_recursive/\<N\>/value | uint64 | Total amount of time between request dispatch and request completion for the IOs done from all the descendant cgroups
+blkio_stats/io_serviced_recursive/\<N\>/value | uint64 | Number of IOs (bio) issued to the disk from all the descendant cgroups
+blkio_stats/io_time_recursive/\<N\>/value | uint64 | Disk time allocated to cgroup per device in milliseconds from all the descendant cgroups
+blkio_stats/io_wait_time_recursive/\<N\>/value | uint64 | Total amount of time the IOs for this cgroup spent waiting in the scheduler queues for service from all the descendant cgroups
+blkio_stats/sectors_recursive/\<N\>/value | uint64 | number of sectors transferred to/from disk from all descendant group
 
 <sup>(1)</sup> Hierarchical version of cgroups counter which in addition to the cgroup's own value includes the sum of all hierarchical children's values
 
```

Comment:
Would you like to update the link to Kernel cgroups documentation, currenty it does not work.
So line https://github.com/intelsdi-x/snap-plugin-collector-docker/pull/65/files#diff-0c2fed19b8992e0dd561b08b21c01903R131 should be changed to:
`Read more about cgroups in [Kernel documentation](https://www.kernel.org/doc/Documentation/cgroup-v1/cgroups.txt)`